.github/ISSUE_TEMPLATE/frontend-issue-template.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Frontend issue template
```

Comment:
Do we need a separate template for fronend or it can be done via usual issue template?